com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: tuxpusher
+    buildsystem: simple
+    build-commands:
+      - cd TuxPusher_SDL;make;make PREFIX=${FLATPAK_DEST} install
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.desktop ${FLATPAK_DEST}/share/applications/tuxpusher.desktop
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.appdata.xml ${FLATPAK_DEST}/share/metainfo/tuxpusher.appdata.xml
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/tuxpusher.png
+    sources:
+      - type: git
+        url: https://github.com/mrbid/TuxPusher.git
```

Comment:
you need a specific commit or tag here.

Reply:
i added a tag and the tag commit hash now, but I only need to provide one or the other not both right?